scripts/windows_dev_setup.ps1

```diff
@@ -336,17 +360,133 @@ function existing_package {
 	Write-Host "This package is already installed."
 }
 
-welcome_message
-verify_architecture
-check_os
-install_msvc_build_tools
-install_llvm
-install_openssl
-install_nodejs
-install_pnpm
-install_postgresql
-install_python
-install_protoc
-install_rustup
-install_cargo_plugins
+function install_git {
+  if (Get-Command git -ErrorAction SilentlyContinue) {
+    Write-Host "Installing Git..."
+    winget install Git.Git --silent
+  } 
+  else {
+    winget upgrade --id Git.Git
+  }
+}
+
+function install_cvc5 {
+  if (![System.IO.Path]::IsPathRooted($env:CVC5_EXE)) {
+    Write-Host "Installing CVC5..."
+    # Download and extract the 64-bit version of Protoc
+    $cvc5_exe = "$env:USERPROFILE\cvc5-$global:cvc5_version\cvc5-Win64.exe"
+    Write-Host $path
+    Invoke-WebRequest -Uri "https://github.com/cvc5/cvc5/releases/download/cvc5-$global:cvc5_version/cvc5-Win64.exe" -OutFile (New-Item -Path "$cvc5_exe" -Force) -ErrorAction SilentlyContinue
+    [Environment]::SetEnvironmentVariable("CVC5_EXE", "$cvc5_exe", "User") 
+    Write-Host "User environment variables set for CVC5"
+  }
+  elseif ($global:architecture -eq "86") {
+    Write-Host "Only 64-bit systems can install CVC5"
+  }

```

Comment:
Do you want to check this first that it's not 64 bit then, check the path rooted?